ras_party/views/enrolments_view.py

```diff
@@ -0,0 +1,60 @@
+import logging
+
+import structlog
+from flask import Blueprint, Response, current_app, make_response, request
+from flask_httpauth import HTTPBasicAuth
+from sqlalchemy.exc import DataError
+from sqlalchemy.orm.exc import NoResultFound
+from werkzeug.exceptions import BadRequest, NotFound
+
+from ras_party.controllers.enrolments_controller import enrolments_by_parameters
+
+logger = structlog.wrap_logger(logging.getLogger(__name__))
+enrolments_view = Blueprint("enrolments_view", __name__)
+auth = HTTPBasicAuth()
+
+
+@enrolments_view.before_request
+@auth.login_required
+def before_respondent_view():
+    pass
+
+
+@auth.get_password
+def get_pw(username):
+    config_username = current_app.config["SECURITY_USER_NAME"]
+    config_password = current_app.config["SECURITY_USER_PASSWORD"]
+    if username == config_username:
+        return config_password
+
+
+@enrolments_view.route("/enrolments", methods=["GET"])
+def get_enrolments() -> Response:
+    json = request.get_json()
+    party_uuid = json.get("party_uuid")
+    business_id = json.get("business_id")
+    survey_id = json.get("survey_id")
+    status = json.get("status")
+
+    if not (party_uuid or business_id or survey_id):
+        logger.error("No parameters passed to get_enrolments")
```

Comment:
I will change this one, I don't like the error message

Reply:
I guess it gets the point across but maybe something that tells us what we could pass?